pkg/builder/example_test.go

```diff
@@ -40,6 +41,8 @@ import (
 //
 // * Start the application.
 func ExampleBuilder() {
+	logf.SetLogger(zap.New())
+	
```

Comment:
Nit: There is a stale tab here in the otherwise empty line (interesting that github doesn't show that), can you remove that? Other than that the PR looks good.